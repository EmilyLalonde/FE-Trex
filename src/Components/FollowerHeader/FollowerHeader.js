import React, { Component } from 'react';
import { 
  StyleSheet, 
  Text, 
  View, 
  TouchableOpacity,
  SafeAreaView
} from 'react-native';

export default class Header extends Component {
  constructor(props) {
    super(props)
  }

  render() {
    return(
      <>
      <SafeAreaView style={styles.top}/>
      <Text style={styles.header}>Trex</Text>
      </>
    )
  }
}

const styles = StyleSheet.create({
  top: {
    backgroundColor: '#84183B',
    flex: 0
  },
  header: {
    backgroundColor: '#84183B',
    color: 'white',
    paddingLeft: 10,
    textAlign: 'center',
    fontSize: 50,
    height: 60,
  }
})