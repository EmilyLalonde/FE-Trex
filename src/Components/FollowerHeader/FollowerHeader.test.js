import React from "react";
import "react-native";
import FollowerHeader from "./FollowerHeader";
import { shallow } from "enzyme";

describe("FollowerHeader", () => {
  it("should return true", () => {
    expect(true).toEqual(true);
  });
  it("should match the snapshot when loading", () => {
    let wrapper = shallow(<FollowerHeader navigation={{ navigate: jest.fn() }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
