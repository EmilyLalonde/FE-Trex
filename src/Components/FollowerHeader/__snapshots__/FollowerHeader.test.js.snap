// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FollowerHeader should match the snapshot when loading 1`] = `
<Fragment>
  <SafeAreaView
    style={
      Object {
        "backgroundColor": "#84183B",
        "flex": 0,
      }
    }
  />
  <Text
    style={
      Object {
        "backgroundColor": "#84183B",
        "color": "white",
        "fontSize": 50,
        "height": 60,
        "paddingLeft": 10,
        "textAlign": "center",
      }
    }
  >
    Trex
  </Text>
</Fragment>
`;
